implementations/rust/ockam/ockam_command/src/project/util.rs

```diff
@@ -62,12 +62,20 @@ pub async fn get_projects_secure_channels_from_config_lookup(
     // Create a secure channel for each project.
     for name in meta.project.iter() {
         // Get the project node's access route + identity id from the config
-        let (project_access_route, project_identity_id) = {
+        let (project_access_route, project_identity) = {
```

Comment:
I know it may look confusing, but the correct term here is `project_identity_id`, so you should revert this rename here.